source/blog/2015-06-17-ember-data-1-13-released.md

```diff
@@ -0,0 +1,376 @@
+
+```js
+  /**
+    @method shouldReloadRecord
+    @param {DS.Store} store
+    @param {DS.Snapshot} snapshot
+    @return {Boolean}
+  */
+  shouldReloadRecord: function(store, snapshot),
+```
+
+`Adapter#shouldBackgroundReloadRecord` is called by the store when the store is resolving the `findRecord` promise with a cached record. If this method returns `true` the store will attempt to reload the record from the adapter in the background. The default implementation for this method in Ember Data 1.13 is to always return `false`. However in Ember Data 2.0 this will be changed to always return `true`.
+
+```js
+  /**
+    @method shouldBackgroundReloadRecord
+    @param {DS.Store} store
+    @param {DS.Snapshot} snapshot
+    @return {Boolean}
+  */
+  shouldBackgroundReloadRecord: function(store, snapshot),
+```
+
+Symmetric methods have also been added for `store.findAll`.
+
+`Adapter#shouldReloadAll` is called by the store after `store.findAll`is called. If it returns true the store will reload the record from the backend and it will wait to resolve the promise until the reload is complete. `shouldReloadAll` is never checked if the record is not in the store or if the user passed `{ reload: true}` in the options argument to `findAll`. The default implementation for this method in Ember Data 1.13 is to always return `true`. However in Ember Data 2.0 this will be changed to return `true` when there are records in the store of the requested type.
+
+```js
+  /**
+    @method shouldReloadAll
+    @param {DS.Store} store
+    @param {DS.SnapshotRecordArray} snapshotRecordArray
+    @return {Boolean}
+  */
+  shouldReloadAll: function(store, snapshotRecordArray),
+```
+
+`Adapter#shouldBackgroundReloadAll` is called by the store when the store is resolving the `findAll` because `shouldReloadAll` returned true. If this method returns `true` the store will attempt to reload the record from the adapter in the background. The default implementation for this method in both Ember Data 1.13 and Ember Data 2.0 is to always return `true`.
+
+```js
+  /**
+    @method shouldBackgroundReloadAll
+    @param {DS.Store} store
+    @param {DS.SnapshotRecordArray} snapshotRecordArray
+    @return {Boolean}
+  */
+  shouldBackgroundReloadAll: function(store, snapshotRecordArray),
+```
+
+### JSON-API Support
+
+Ember Data 1.13 comes with support for [JSON-API](http://jsonapi.org/) 1.0. There is a new `DS.JSONAPIAdapter` and `DS.JSONAPISerializer` that support working with JSON-API backends. In Ember Data 2.0 the JSON-API adapter and serializer will be the default adapter loaded by Ember Data.
```

Comment:
We should add a note that JSONAPISerializer only supports the new Serializer API.